src/main/resources/GoogleApps/Translations.fr.xml

```diff
@@ -39,17 +39,27 @@
   <minorEdit>false</minorEdit>
   <syntaxId>plain/1.0</syntaxId>
   <hidden>false</hidden>
-  <content>admin.googleapps=Google Apps
+  <content><![CDATA[
```

Comment:
Hi @polx! 
There's no point in using `<![CDATA[]]` in translation files as they should use only plain text (not HTML, nor XWiki syntax). We should use parametrized translations, described in https://extensions.xwiki.org/xwiki/bin/view/Extension/Localization/Scripting/, and so the translation process to be as easy as possible for the end users/contributors. At the same time information like URLs should be handled in the code, not in translations.
Ex for a translation with url:
`googleapps.config.explanation=XWiki peut-etre intégré avec Google Apps, ce qui permet ensuite l''édition..{0}enregistrer pour l'accès à OAuth sur Google.com{1}`
The usage in a wiki page woule be:
`$services.localization.render('googleapps.config.explanation', ['[[', '>>path:https://accounts.google.com/ManageDomains]]'])`

The only thing to care about is to double the `'` char from `l'édition` (to obtain `l''édition`) when the translation contains parameters (there's something to fix about this, but that's the Localization module's job).